### PR TITLE
[flux-core v0.11] doc: Update README.md in example5/

### PR DESCRIPTION
**Pull request by cmoussa1**
Oops, I'm sorry! When submitting all of these PR's I must have forgotten to submit this specific example. 

This PR makes the following changes to **README.md**:

- Added numbers to instructions
- A table listing of the equivalent commands for setting FLUX_SCHED_OPTIONS in both tcsh and bash/zsh shells.
- Added code format to step-by-step instructions
- Changed allocation for `io-forwarding.lua` script 
- Added a note to user to inform them that output might not show up until all tasks complete.

### CODE REVIEW

**Comment by dongahn** [comment]
No problem. Don't you need PKG_CONF_PATH before make though?

**Comment by cmoussa1** [comment]
You're right; I think I must have made those commits before you and I sat down and fully debugged the issues for walking through this workflow. Added the `PKG_CONFIG_PATH` instruction to **README.md**

**Comment by cmoussa1** [comment]
Force-pushed some needed changes to **README.md**; I believe this is ready for another review.

File: example5/README.md [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+`export FLUX_MODULE_PATH=${FLUX_MODULE_PATH}:$(pwd)`
+
+5. Make sure the scheduler module will do node-exclusive scheduling:
+
+| Shell     | Command                                        |
+| -----     | ----------                                     |
+| tcsh      | `setenv FLUX_SCHED_OPTIONS "node-excl=true"`   |
+| bash/zsh  | `export FLUX_SCHED_OPTIONS='node-excl=true'`   |
+
+<<<<<<< HEAD
```
Comment (dongahn) [inline example5/README.md]: Merge errors?

Comment (cmoussa1) [reply example5/README.md]: Ahh! I've been wondering where I went wrong. I think I must have made an error when rebasing a couple of commits to fix the `PKG_CONFIG_PATH` instructions in **README.md**.

File: example5/README.md [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+| tcsh      | `setenv FLUX_SCHED_OPTIONS "node-excl=true"`   |
+| bash/zsh  | `export FLUX_SCHED_OPTIONS='node-excl=true'`   |
+
+6. `srun --pty --mpi=none -N3 flux start -o,-S,log-filename=out`
+
+7. `flux submit -N 2 -n 2 ./compute.lua 120`
+
+8. `flux submit -N 1 -n 1 ./io-forwarding.lua 120`
+
+__NOTE: Due to this version of Flux, you may not see output of your jobs until
```
Comment (dongahn) [inline example5/README.md]: My guess is that if you add that Lua statement that makes stdout unbufferred, this stdout issue will be fixed and you won't need this Note?

Comment (cmoussa1) [reply example5/README.md]: Yeah, I agree. The output from attaching to `compute.lua` comes right away, so we probably don't need this Note anymore. I'll make this change and re-ping you once it's completed.

**Comment by dongahn** [comment]
@cmoussa1: Is this ready to go in? Or do you want to add a commit that make stdout unbuffered as well?

**Comment by cmoussa1** [comment]
I forgot that buffered I/O was also prevalent in `flux-core v0.11`; the commit that makes that change in **compute.lua** and **io-forwarding.lua** has been added!

**Comment by dongahn** [comment]
LGTM. Merging.

**Comment by cmoussa1** [comment]
Thank you!

**Pull request merged**